shell.nix

```diff
@@ -0,0 +1,22 @@
+let
+  pkgs = import <nixpkgs> {};
+
+  my-python-packages = python-packages: with python-packages; [
+    # Dependencies
+    aiohttp
```

Comment:
These will go out of date as the project evolves if none of the core developers use nix.

Reply:
Indeed, I'm a bit hesitant to add distribution specific packaging.

Reply:
This is merely a development environment, and not a full package. But I can remove it if you prefer to.

@andir Do you know of a way to automatically synchronize the dependencies without manual intervention?

Reply:
Yes, just use a virtualenv for your development. It will also ensure you are using the same versions as the project describes. Obviously you can go all in and use some random 2nix tooling but that is yet again burden on the core developers of this project.

Reply:
Virtualenv is a good point. However, I don't think one needs Nix for that, those three commands could simply directly go in `.envrc`. I'll try that.

Reply:
Can you please remove `.envrc` completely? If you need it locally you can add it to your git exclude list but I don't think it belongs in this repository. Thanks.

Reply:
Of course.